docs/sphinx/source/whatsnew/v0.11.1.rst

```diff
@@ -109,3 +101,11 @@ Contributors
 * Eduardo Sarquis (:ghuser:`EduardoSarquis`)
 * Adam R. Jensen (:ghuser:`AdamRJensen`)
 * Andrew B Godbehere (:ghuser:`agodbehere`)
```

Comment:
while the Muneer PR has not been merged yet, @BernatNicolau has still made other contributions (issues and merged PRs)
```suggestion
* Andrew B Godbehere (:ghuser:`agodbehere`)
* Bernat Nicolau (:ghuser:`BernatNicolau`)
```

Reply:
@BernatNicolau already is in the whatsnew, just outside the diff. Source: https://github.com/kandersolar/pvlib-python/blob/82e47829fb5390887f6ad07778a982f9fa14decb/docs/sphinx/source/whatsnew/v0.11.1.rst?plain=1#L100